server/server.go

```diff
@@ -23,32 +25,49 @@ const (
 	Version = "0.0.1"
 )
 
-type Mode handler.Mode
+type Mode int
 
 const (
-	DEV Mode = iota
-	PROD
+	ModeProd Mode = iota
+	ModeDev
 )
 
+func (m Mode) IsProd() bool {
+	switch m {
```

Comment:
Can't we just do `return m.Mode == ModeProd` instead of this switch case?

Reply:
Yes, it is a simple case, and I don't see it evolve that much.

Reply:
done